src/server/main_service.h

```diff
@@ -203,4 +203,8 @@ class Service : public facade::ServiceInterface {
   uint32_t loading_state_counter_ ABSL_GUARDED_BY(mu_) = 0;
 };
 
+void ArmShutdownWatchdog(util::ProactorPool& pp);
+
+void DisarmShutdownWatchdog();
```

Comment:
right now, the disarm method does not need to be part of the public api. it can be a static method in the anonymous namespace in `.cc` compilation unit for main service. But I added this here as it makes it possible to disarm the timer from outside if needed eg from `dfly_main`

Reply:
removed these as no longer needed after move to main service